Content/posts/building-3238-packages-for-apple-silicon.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-08-20 12:00
+title: Building 3,238 Swift Packages for Apple Silicon
+description: As part of the Swift Package Index build system, we have processed what must be the most extensive test of Apple Silicon compatibility outside of Apple. Here's what we found.
+---
+
+As part of the [Swift Package Index build system](https://blog.swiftpackageindex.com/posts/launching-language-and-platform-package-compatibility/), we have processed what must be the most extensive test of Apple Silicon compatibility outside of Apple.
+
+At the time of writing, we currently have XX,XXX Apple Silicon builds in our database covering 3,238 packages. Why so many? We test up to [three versions of each package](https://swiftpackageindex.com/SwifQL/SwifQL/builds) and compile each with both `xcodebuild` and `swift build`.
```

Comment:
This number needs updating, do you have the query handy?

Reply:
```
select count(*)
from builds
where platform in ('macos-spm-arm', 'macos-xcodebuild-arm')
and status = 'ok'
```

8291 as of this morning

Reply:
Fixed. I put in 12,942 as the total number of builds as this query only lists successful builds, and at this point we're talking about total.